### PR TITLE
Add new test to catch sequential names

### PR DESCRIPTION
**Pull request by matthewmorgan**
As discussed in #177

### CODE REVIEW

File: robot-name/robot-name.spec.js [hunk 1]
```diff
@@ -60,4 +60,33 @@ describe('Robot', () => {
...
 
+
+  xit('should test for sequential names', () => {
+    const name1 = robot.name;
+    robot.reset();
+    const name2 = robot.name;
+    robot.reset();
+    const name3 = robot.name;
+
+    expect(areSequential(name1, name2)).not.toBe(true);
```
Comment (rchavarria) [inline robot-name/robot-name.spec.js]: You can expect falsy values in different ways in Jasmine.

```
expect(areSequential(name1, name2)).not.toBe(true);
expect(areSequential(name1, name2)).toBe(false);
expect(areSequential(name1, name2)).toBeFalsy();
```

I'm not very fan of the first one. What do you think? What do you think is the most readable?

**Comment by matthewmorgan** [comment]
I had to look up the syntax on `not.toBe()` myself.  I have seen it used and wanted to try it!

But I don't really feel strongly about it-- option 2 would be fine with me otherwise.

**Comment by rchavarria** [comment]
Ok, let's use second option.

Another question to generate names. What about generate them this way?

```
const name1 = robot.name; // the first robot instance is already generated
const name2 = new Robot().name;
const name3 = new Robot().name;
```

**Comment by matthewmorgan** [comment]
I propose we test both `Robot.reset().name` and `new Robot().name`.  In the event someone were using different techniques to generate vs resetting, we would want to test both for non-sequential output, right?

**Comment by rchavarria** [comment]
I see your point, some users can create and reset names differently.

So, let's create then two tests: one to check new names, one to check reset names. Both tests should look for non-sequential names. Do you agree?

**Comment by matthewmorgan** [comment]
done!

**Comment by rchavarria** [comment]
There is a new commit because I rebased it onto master, but I changed no code.

Thanks a lot for this code. Those new tests look awesome!!

**Pull request merged**